src/WireMock.Net.Aspire/WireMockServerLifecycleHook.cs

```diff
@@ -10,32 +10,38 @@ internal class WireMockServerLifecycleHook(ILoggerFactory loggerFactory) : IDist
 {
     private readonly CancellationTokenSource _shutdownCts = new();
 
-    public async Task AfterResourcesCreatedAsync(DistributedApplicationModel appModel, CancellationToken cancellationToken = default)
+    private Task? _mappingTask;
+
+    public async Task AfterEndpointsAllocatedAsync(DistributedApplicationModel appModel, CancellationToken cancellationToken = default)
     {
         var cts = CancellationTokenSource.CreateLinkedTokenSource(_shutdownCts.Token, cancellationToken);
 
-        var wireMockServerResources = appModel.Resources
-            .OfType<WireMockServerResource>()
-            .ToArray();
-
-        foreach (var wireMockServerResource in wireMockServerResources)
+        _mappingTask = Task.Run(async () =>
```

Comment:
maybe use SafeFireAndForget (https://github.com/TheCodeTraveler/AsyncAwaitBestPractices) or something similar to indicate that this is a long running task and exceptions do not matter or are only logged?

Reply:
I don't think pulling in a third party library is worth it for such a small thing, but I could add a comment explaining the intention.

Any exceptions thrown in the task will be re-thrown in the `DisposeAsync` method.